markhor_tracks/include/markhor_hw_interface.hpp

```diff
@@ -69,6 +69,8 @@ class MarkhorHWInterface : public hardware_interface::RobotHW
 
   double tracks_fb_coeff = 1;
 
+  int allowable_closedloop_error = 1;
```

Comment:
Should be 0

Reply:
Isn't there a reason it's not 0? Like some drive weirdness where it just makes it work all the time?

Reply:
As this is a velocity driven control loop, allowing closed loop error means we allow the track to go to a certain speed until it fights movement, this is not the desired behaviour as we need to prevent slippage in the stairs for better control. In practice, this is a speed of 1 encoder step/100ms, which would be unnoticeable but this might become an issue later, let's do it right

Reply:
I'll try to test it tomorrow if I have time just to be sure it doesn't create any issues and I will update this PR accordingly